configs/train_model/2021.02.21.ini

```diff
@@ -0,0 +1,20 @@
+[DEFAULT]
+version = 2021.02.21
+description = Train all models with 210221 training data.
```

Comment:
generally I think it is better to have config per model, even though I understand that in this case since you are ensembling it possibly makes sense to be able to train multiple models. what would possibly be better in my mind is to have defined a VotingClassifier or WellcomeVotingClassifier that defines the models used and trains those.

I am saying that only because in my mind each config is related to a model and this could be replicated for ensemble models either by training models separately or defining all the models that the ensemble model needs as params of using a custom model or something like VotingClassifier.

This is not a hard rule of course, just a preference.